src/app/components/standards/test-tool/test-tool-edit.jsx

```diff
@@ -41,6 +43,11 @@ const useStyles = makeStyles({
     gap: '8px',
     flexWrap: 'wrap',
   },
+  dataContainer: {
```

Comment:
I don't like this name. If it's a data container, then why aren't the dropdowns for rule & criteria in one of these containers? Would "horizontalInput" be closer to functionally describing what they do?